.github/workflows/main.yml

```diff
@@ -0,0 +1,69 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Poliastro Website CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the sources branch
+  push:
+    branches: 
+      - sources
+      - 'content'
```

Comment:
I wonder why the second branch name needs quotes?

Reply:
I have tried to implement it, in such a way that if any changes to the `content` folder of the repository in any branch (PR based or else wise) are made, the pipeline will trigger the build execution (for the same please check this [link](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags))

Reply:
The link you sent refers to branches. I think this is what you mean:

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-paths

In any case, let's remove the `'content'` line (and the other below this one) so we only trigger builds on the `sources` branch and pull requests.